script.js

```diff
@@ -131,6 +131,21 @@ const createListItem = (location, status, lng, lat) => {
   if (location.seekingVolunteers && location.seekingVolunteers.match(/(?:\byes\b)/i)) {
     seekingVolunteers = `<span class="seekingVolunteers location-list--badge">Needs Volunteer Support</span>`
   }
+
+  const openTimeDistribution = moment(location.openingForDistributingDontations, ["h:mm A"])
+  const openTimeDistributionLessOne = moment(location.openingForDistributingDontations, ["h:mm A"]).subtract(1, 'hours')
+  let openingSoonForDistribution = ''
+  if (moment().isBetween(openTimeDistributionLessOne, openTimeDistribution)) {
+    openingSoonForDistribution = `<p class="opening-soon">Opening soon! ${openTimeDistribution.format("LT")} for distributing</p>`
+  }
+
+  const openTimeReceiving = moment(location.openingForReceivingDontations, ["h:mm A"])
+  const openTimeReceivingLessOne = moment(location.openingForReceivingDontations, ["h:mm A"]).subtract(1, 'hours')
+  let openingSoonForReceiving = ''
+  if (moment().isBetween(openTimeReceivingLessOne, openTimeReceiving)) {
+    openingSoonForReceiving = `<p class="opening-soon">Opening soon! ${openTimeReceiving.format("LT")} for receiving</p>`
```

Comment:
Same translation comment...
```suggestion
    openingSoonForReceiving = `<p class="opening-soon"><span data-translation-id="opening_soon">Opening soon!</span> ${openTimeReceiving.format("LT")} <span data-translation-id="for_receiving">for receiving</span></p>`
```

Reply:
I just added `opening_soon`, `for_distribution` and `for_receiving` to the translation spreadsheet, assuming we'll use those identifiers.

Reply:
Updated! Kept the suggested identifiers!